src/test/java/com/application/bookingservice/repository/payment/PaymentRepositoryTest.java

```diff
@@ -0,0 +1,91 @@
+package com.application.bookingservice.repository.payment;
+
+import static com.application.bookingservice.model.Payment.Status;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.application.bookingservice.model.Booking;
+import com.application.bookingservice.model.Payment;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.jdbc.Sql;
+
+@DataJpaTest
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+class PaymentRepositoryTest {
+    private static Payment payment;
+    @Autowired
+    private PaymentRepository paymentRepository;
+
+    @BeforeAll
+    static void beforeAll() {
+        Booking booking = new Booking();
+        booking.setId(1L);
+
+        payment = new Payment();
+        payment.setId(1L);
+        payment.setBooking(booking);
+        payment.setStatus(Status.SUCCEED);
+        payment.setTotal(new BigDecimal("100.00"));
+        payment.setSessionUrl("Example");
+        payment.setSessionId("hirsdHS_tjsjsyyjstIT");
+    }
+
+    @Test
+    @DisplayName("Find payment by session id")
+    @Sql(scripts = {
+            "classpath:db/address/add-addresses.sql",
+            "classpath:db/accommodation/add-accommodations.sql",
+            "classpath:db/booking/add-bookings.sql",
+            "classpath:db/payment/add-payments.sql"
+    }, executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)

```

Comment:
move this scripts to the `BeforeEach()` method, because you have such fields in the another method in this class